plugins/out_s3/s3.c

```diff
@@ -2470,6 +2476,11 @@ static struct flb_config_map config_map[] = {
      "will be stored with the default 'STANDARD' storage class."
     },
 
+         {
+          FLB_CONFIG_MAP_STR, "profile", NULL,
+          0, FLB_TRUE, offsetof(struct flb_s3, profile),
+          "AWS profile"
+    },
```

Comment:
add this using FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP ?

Reply:
Done.

Reply:
I changed my mind we don't need to use FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP

Because it doesn't have the `offsetof` part so its less convenient.